src/pages/reports/index/Reports.tsx

```diff
@@ -239,18 +239,12 @@ export default function Reports() {
             setIsPendingExport(false);
           });
       })
-      .catch((error: AxiosError<ValidationBag | Blob>) => {
+      .catch((error: AxiosError<ValidationBag>) => {
         if (error.response?.status === 422) {
-          if (report.payload.send_email) {
-            setErrors(error.response.data as ValidationBag);
-          }
-
-          if (!report.payload.send_email) {
-            const blob = error.response.data as Blob;
-
-            blob.text().then((text) => setErrors(JSON.parse(text)));
```

Comment:
@beganovich I'm not sure I can find a case where resolving the blob is needed and then putting it into an errors state. Whatever I did, I got the standard error object from the API, and this handling causes an issue with not displaying the message. Please let me know if there is a case where this is still needed.

Reply:
I am sure it was set with a reason. If it still works without, then we good.

Reply:
@beganovich I just retested it, and the behavior is exactly the same. As I can see here, error handling is separated based on the `send_email` boolean value, which I tested, and the response from the API is not different, so it does not require different handling. I believe that this approach is correct as a solution.